quickwit-indexing/src/actors/indexer.rs

```diff
@@ -239,11 +257,16 @@ impl IndexerState {
     }
 }
 
+struct IndexingWorkbench {
```

Comment:
What an indexing workbench is is not exactly intuitive. Might be worth adding a comment in the follow-up PR that handles multiple splits in the workbench.